### PR TITLE
func_80240790_BBD930

### PR DESCRIPTION
**Pull request by kklions1**

### CODE REVIEW

File: src/world/area_obk/obk_01/BBD1A0.c [hunk 1]
```diff
@@ -21,4 +21,11 @@ INCLUDE_ASM(s32, "world/area_obk/obk_01/BBD1A0", func_802406AC_BBD84C);
 
 INCLUDE_ASM(s32, "world/area_obk/obk_01/BBD1A0", func_80240730_BBD8D0);
 
-INCLUDE_ASM(s32, "world/area_obk/obk_01/BBD1A0", func_80240790_BBD930);
+ApiStatus func_80240790_BBD930(Evt* script, s32 isInitialCall) {
+    ItemEntity* temp_s0;
+
+    temp_s0 = get_item_entity(script->varTable[0]);
```
Comment (nanaian) [inline src/world/area_obk/obk_01/BBD1A0.c:27]: You can put this on one line like
```suggestion
ApiStatus func_80240790_BBD930(Evt* script, s32 isInitialCall) {
    ItemEntity* temp_s0 = get_item_entity(script->varTable[0]);

```

**Review by nanaian — CHANGES_REQUESTED** [review]
Nice match! Just a small style suggestion

File: src/world/area_obk/obk_01/BBD1A0.c [hunk 2]
```diff
@@ -21,4 +21,11 @@ INCLUDE_ASM(s32, "world/area_obk/obk_01/BBD1A0", func_802406AC_BBD84C);
 
 INCLUDE_ASM(s32, "world/area_obk/obk_01/BBD1A0", func_80240730_BBD8D0);
 
-INCLUDE_ASM(s32, "world/area_obk/obk_01/BBD1A0", func_80240790_BBD930);
+ApiStatus func_80240790_BBD930(Evt* script, s32 isInitialCall) {
+    ItemEntity* temp_s0;
```
Comment (ethteck) [inline src/world/area_obk/obk_01/BBD1A0.c:25]:
```suggestion
    ItemEntity* temp_s0 = get_item_entity(script->varTable[0]);
```
(followed by empty line)

**Comment by BowserSlug** [comment]
🚀 This PR matches 1 function (+0.00%) on `us`.

File: src/world/area_obk/obk_01/BBD1A0.c [hunk 3]
```diff
@@ -21,4 +21,10 @@ INCLUDE_ASM(s32, "world/area_obk/obk_01/BBD1A0", func_802406AC_BBD84C);
 
 INCLUDE_ASM(s32, "world/area_obk/obk_01/BBD1A0", func_80240730_BBD8D0);
 
-INCLUDE_ASM(s32, "world/area_obk/obk_01/BBD1A0", func_80240790_BBD930);
+ApiStatus func_80240790_BBD930(Evt* script, s32 isInitialCall) {
+    
+    ItemEntity* item = get_item_entity(script->varTable[0]);
+    set_item_entity_flags(script->varTable[0], 0x80000);
```
Comment (nanaian) [inline src/world/area_obk/obk_01/BBD1A0.c:27]:
```suggestion
    ItemEntity* item = get_item_entity(script->varTable[0]);    

    set_item_entity_flags(script->varTable[0], 0x80000);
```

**Review by nanaian — CHANGES_REQUESTED** [review]

**Pull request merged**